### PR TITLE
Cockpit 202

### PR DESCRIPTION
**Pull request by KKoukiou**
Closes #273

Rendered preview https://kkoukiou.github.io/cockpit-project.github.io/blog/cockpit-202.html

### CODE REVIEW

File: _posts/2019-09-04-cockpit-202.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+summary: Cockpit with Machines improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly.  Here are the release notes from version 202.
+
+### Machines: Creation of Storage Volumes
+
+Creation of Storage Volumes is now possible for a subset of the supported Storage Pool types.
+The Storage Volumes can then be used a virtual disks for VMs.
```
Comment (allisonkarlitskaya) [inline _posts/2019-09-04-cockpit-202.md]: used *as virtual disks

File: _posts/2019-09-04-cockpit-202.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+![VM Storage Volume creation](/images/create-storage-volume.png)
+
+### Improved component for inserting paths on the filesystem
```
Comment (allisonkarlitskaya) [inline _posts/2019-09-04-cockpit-202.md]: I feel like you want "selecting", not "inserting".

File: _posts/2019-09-04-cockpit-202.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+The component used in dialogs for inserting paths on the filesystem was improved in order to be keyboard navigatable and accessible.
+Multiple dialogs such as SSH key authentication from the Shell, Create VM and Create Storage Pool from Machines Page benefited from this change.
```
Comment (allisonkarlitskaya) [inline _posts/2019-09-04-cockpit-202.md]: s/benefited/benefit/

**Comment by KKoukiou** [comment]
Please don''t merge - I need to update the bodhi URL once I have it.

**Pull request merged**